db/commitlog/commitlog.cc

```diff
@@ -1741,41 +1741,64 @@ future<> db::commitlog::segment_manager::delete_file(const sstring& filename) {
 }
 
```

Comment:
Please add a description of the fix to the changelog, so I don't have to reverse-engineer it.

Reply:
Ok.